salt/kubelet/remove-pre-reboot.sls

```diff
@@ -0,0 +1,22 @@
+# cleanup all the things we have created
+
+{%- set name = 'node:' + grains['caasp_fqdn'] %}
```

Comment:
`caasp_fqdn`? I thought we don't use that anymore...

Reply:
Ok, it seems that in meantime `nodename` PR was merged. Thank you I fixed that